src/includes/class-wordlift.php

```diff
@@ -1713,7 +1717,8 @@ private function define_admin_hooks() {
 		$this->loader->add_action( 'wp_ajax_wl_relation_rebuild_process_all', $this->relation_rebuild_adapter, 'process_all' );
 		$this->loader->add_action( 'wp_ajax_wl_sample_data_create', $this->sample_data_ajax_adapter, 'create' );
 		$this->loader->add_action( 'wp_ajax_wl_sample_data_delete', $this->sample_data_ajax_adapter, 'delete' );
-
+		$excerpt_adapter = new Post_Excerpt_Meta_Box_Adapter();
+		$this->loader->add_action('do_meta_boxes', $excerpt_adapter, 'replace_post_excerpt_meta_box');
```

Comment:
@naveen17797 initially we'll release the features only for web sites using English (see `Wordlift_Configuration_Service` to get the configured language). Please add a condition here.